packages/ui/__tests__/CheckboxFormik.test.tsx

```diff
@@ -43,5 +43,40 @@ describe('CheckboxFormik', () => {
       },
       expect.anything(),
     )
+    expect(wrapper.find('input').props()).toHaveProperty('checked')
+  })
+
+  it('Checkbox is changed in a formik scenario', async () => {
```

Comment:
nit: SHould we combine it with a previous test like we do for some other components?

Reply:
@aigoncharov Since it's nit, I'll do that in a separate PR if it's OK, we'd like to get this merged soon, because we need it on .cloud